RubiksCubeSolver/src/main/java/com/mycompany/rubikscubesolver/RubiksCube.java

```diff
@@ -10,99 +10,17 @@ public class RubiksCube {
     // cube[5] is spot number 6
     public static void main(String[] arguments)
     {
-        char[] cube = { 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W',
+        char[] cube = { 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 'W', 
                         'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O',
                         'G', 'G', 'G', 'G', 'G', 'G', 'G', 'G', 'G',
-                        'R', 'R', 'R', 'R', 'R', 'R', 'R', 'R', 'R',
-                        'Y', 'Y', 'Y', 'Y', 'Y', 'Y', 'Y', 'Y', 'Y',
-                        'B', 'B', 'B', 'B', 'B', 'B', 'B', 'B', 'B',
+                        'R', 'R', 'R', 'R', 'R', 'R', 'R', 'R', 'R', 
+                        'Y', 'Y', 'Y', 'Y', 'Y', 'Y', 'Y', 'Y', 'Y', 
+                        'B', 'B', 'B', 'B', 'B', 'B', 'B', 'B', 'B', 
                       };
         
-        String printCube = new String(cube);
```

Comment:
will you fix this please? :) copy the stuff from the main function from the main branch on github.com, so it doesn't try to delete it when i merge the pull request?